tests/foreman/cli/test_settings.py

```diff
@@ -464,35 +463,3 @@ def test_positive_failed_login_attempts_limit(setting_update, target_sat):
     assert target_sat.execute(f'hammer -u {username} -p {password} user list').status == 0
     target_sat.cli.Settings.set({'name': 'failed_login_attempts_limit', 'value': '0'})
     assert target_sat.cli.Settings.info({'name': 'failed_login_attempts_limit'})['value'] == '0'
-
-
-@pytest.mark.parametrize('setting_update', ['login_text'], indirect=True)
-def test_positive_existing_setting_info(setting_update, module_target_sat):
-    """Get setting info for a existing setting
-
```

Comment:
I think this test can stay, while non-existing test added for validation of SAT-20238 can be removed only as its not cherrypicked for 6.18.z yet